nursery/send-sms-on-android.yml

```diff
@@ -0,0 +1,24 @@
+rule:
+  meta:
+    name: send SMS on Android
+    namespace: communication/sms
+    authors:
+      - "@mr-tz"
+    scope: function
+    # att&ck:
+    #   - Mobile::SMS Control [T1582]
+  features:
+    - and:
+      - os: android
+      # ... = (*env)->FindClass(env, "android/telephony/SmsManager");
+      - string: "android/telephony/SmsManager"
+      - optional:
```

Comment:
eventually we'll probably want to factor this out, but let's wait until it's used in a few places first.

Reply:
yeah, probably a good idea to write a few dozen rules and talk to potential users first